plugins/citationFormats/abnt/AbntCitationPlugin.inc.php

```diff
@@ -183,6 +185,46 @@ function smartyPluginUrl($params, &$smarty) {
 
 		return $smarty->smartyUrl($params, $smarty);
 	}
+
+	/**
+	 * @function abnt_date_format Format date taking in consideration ABNT month abbreviations
+	 * @param $arg string
+	 * @return string
+	 */
+	function abntDateFormat($string) {
+		if (is_numeric($string)) {
+			// it is a numeric string, we handle it as timestamp
+			$timestamp = (int)$string;
+		} else {
+			$timestamp = strtotime($string);
+		}
+		$format = "%B %Y";
+		if (String::strlen(strftime("%B", $timestamp)) > 4) {
+			$format = "%b. %Y";
+		}
+
+		return strtolower(strftime($format, $timestamp));
```

Comment:
Better to use String::strtolower; depending on PHP and locale settings, strtolower may not be multibyte-savvy. (Also elsewhere.)